### ISSUE TITLE
FATA[0005] failed to initialize project with version "2"

### ISSUE THREAD

**Issue by camilamacedo86** [operator-framework/operator-sdk#3690]
## Bug Report

**What did you do?**
 operator-sdk init --project-version=2 --domain example.org --license apache2 --owner "The Kubernetes authors"

**What did you expect to see?**
project scaffolded since has the option

**What did you see instead? Under which circumstances?**
```
$ operator-sdk init --project-version=2 --domain example.org --license apache2 --owner "The Kubernetes authors"
Writing scaffold for you to edit...
Get controller runtime:
$ go get sigs.k8s.io/controller-runtime@v0.6.2
Update go.mod:
$ go mod tidy
Running make:
$ make
/Users/camilamacedo/go/bin/controller-gen object:headerFile="hack/boilerplate.go.txt" paths="./..."
go fmt ./...
go vet ./...
go build -o bin/manager main.go
Next: define a resource with:
$ operator-sdk create api
Error: failed to initialize project with version "2": error updating Makefile: unsupported plugin key ""
Usage:
  operator-sdk init [flags]

Examples:
  # Scaffold a project using the apache2 license with "The Kubernetes authors" as owners
  operator-sdk init --project-version=2 --domain example.org --license apache2 --owner "The Kubernetes authors"


Flags:
      --domain string            domain for groups (default "my.domain")
      --fetch-deps               ensure dependencies are downloaded (default true)
  -h, --help                     help for init
      --license string           license to use to boilerplate, may be one of 'apache2', 'none' (default "apache2")
      --owner string             owner to add to the copyright
      --plugins strings          Name and optionally version of the plugin to initialize the project with. Available plugins: ("ansible.sdk.operatorframework.io/v1", "go.kubebuilder.io/v2", "helm.sdk.operatorframework.io/v1")
      --project-version string   project version, possible values: ("2", "3-alpha") (default "3-alpha")
      --repo string              name to use for go module (e.g., github.com/user/repo), defaults to the go package of the current working directory.
      --skip-go-version-check    if specified, skip checking the Go version

Global Flags:
      --verbose   Enable verbose logging

FATA[0006] failed to initialize project with version "2": error updating Makefile: unsupported plugin key "" 
```

**Environment**
* operator-sdk version: 4739c1fc5bfb61db28b1bc52b66a63d9493a1ec2
* go version: go1.13.12

**Comment by joelanford** [comment]
Since this is an issue in the `init` command that results in an immediate failure, I think we could consider the fix for this (whatever it is) a non-breaking bug fix.

**Comment by estroz** [comment]
Closed by #3697

**Comment by camilamacedo86** [comment]
Hi @estroz, 

Should we not address it to 0.19.X as well?
